aspnetcore/includes/signalr-version-notice.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> This document details the latest version of ASP.NET Core SignalR. Looking to get started with an earlier version of SignalR? See the [SignalR 1.x documentation](xref:aspnet/signalr/).
```

Comment:
The PU prefers the more ref style.

This document details the latest version of ASP.NET Core SignalR. ~Looking to get started with an earlier version of SignalR?~ See the [SignalR 1.x documentation](xref:aspnet/signalr/) for the ASP.NET Framework version.

Reply:
Ok. I took it from here https://docs.microsoft.com/en-us/aspnet/core/getting-started
I'll add an issue to change it there too then.